### ISSUE TITLE
Class select menu should start on recon

### ISSUE THREAD

**Issue by blaberry** [NeotokyoRebuild/neo#780]
### Build Info

20241102_14c4282

### Description

The class selection menu starts on assault instead of recon, this will probably mess up people who are used to the parity way, where recon is the first class

### To Reproduce

1. Join a team
2. Pick a character

### Expected behavior

I end up as a recon

### Actual behavior

I end up in assault because I didn't remember that we don't have parity for the class selection screen!!!

### Operating System

- [X] Windows
- [ ] Linux

### Version/Distro

Win 10

### Machine's CPU

Ryzen 9 7950X

### Machine's GPU

RTX 3080 FE

### GPU's driver

Nvidia 556.12

### Build's compiler

MSVC 19.41.34120.0

### Additional context and Screenshots

I don't think this is very important but qol is nice.

**Comment by Agiel** [comment]
Personally I don't mind either way, but it might be worth pointing out that having Assault be the default would probably be more newbie friendly.

**Comment by AdamTadeusz** [comment]
I thought we talked about this already and agreed that assault should be the default

**Comment by AdamTadeusz** [comment]
closed wontfix?

**Comment by blaberry** [comment]
Yeah let's close it, I didn't remember the convo we had where we agreed adding 1-3 as class selection was good enough.
